packages/protocol/thread/thread.ts

```diff
@@ -228,8 +228,8 @@ class _ThreadFront {
           supportsEagerEvaluation: false,
           supportsElementsInspector: true,
           supportsEventTypes: true,
-          supportsNetworkRequests: false,
-          supportsRepaintingGraphics: features.chromiumRepaints ?? false,
+          supportsNetworkRequests: true,
+          supportsRepaintingGraphics: features.chromiumRepaints ?? true,
```

Comment:
Line 231 looks okay. I'm not sure about 232 though. Do we mean to turn repaints on for Chromium recordings by default?

Reply:
Yep. that's the goal